uui-components/src/overlays/Dropdown.tsx

```diff
@@ -440,7 +440,12 @@ function DropdownComponent(props: DropdownProps, ref: React.ForwardedRef<HTMLEle
                 <Portal target={ portalTarget }>
                     <FreeFocusInside>
                         <div
-                            role="dialog"
+                            role={
```

Comment:
why we need to remove  role="dialog" from main menu dropdown, but keep for all other dropdowns?